AnagramTS_/src/anagram.ts

```diff
@@ -11,11 +11,11 @@ export function fillMatchedWords() {
   const inputString: string = sorting("melon");
   const inputStringLength: number = inputString.length;
 
-  const wordListLength:number = words.length;
+  const wordListLength: number = words.length;
   const matchedWordsList: string[] = [];
 
   for (let i = 0; i < wordListLength; i++) {
-   // console.log("I:"+i);
+    // console.log("I:"+i);
     if (inputStringLength == words[i].length) {
```

Comment:
in JS/TS always use the triple operators. please read the difference and give an example of a possible problem